### PR TITLE
Feature: basic file save and open with notifications

### PR DESCRIPTION
**Pull request by rgerstenberger**
This adds save and open functionality for osx with `.json` files. More work is needed for windows and supporting a different file extension.

/cc @mdaxtman @kenwheeler

### CODE REVIEW

File: app/containers/home.js [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
 import defaultTheme from "../themes/default";
+import { fileActions } from "../menu-actions";
 import styles from "./home.css";
 import { BLACKLIST_CURRENT_ELEMENT_DESELECT } from "../constants";
 
-const store = new SlidesStore();
+const fileStore = new FileStore();
+const slideStore = new SlidesStore(fileStore);
+
+
```
Comment (mdaxtman) [inline app/containers/home.js]: do we want this extra white space?

Comment (rgerstenberger) [inline app/containers/home.js]: Nope! Good catch :)

File: app/app.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Spectacle Editor</title>
+    <title>Untitled</title>
```
Comment (kenwheeler) [inline app/app.html]: ?

**Comment by kenwheeler** [comment]
LGTM

**Comment by mdaxtman** [comment]
Everything Looks good. 👍

**Pull request merged**